sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -1296,6 +1310,27 @@ public class TestCrossEntityTransactionBatch
             }
         }
 
+        public class TestCustomErrorHandler
+        {
+            public static async Task RunAsync(
+                [ServiceBusTrigger(FirstQueueNameKey)]
+                ServiceBusReceivedMessage message)
+            {
+                // Delay so that the lock will be lost when trying to complete automatically.
+                // We can't simply throw an exception here as it would be swallowed by TryExecuteAsync call in the listener.
+                // This means that the exception handler will not be used for errors originating from the function. This is the same
+                // behavior as in V4.
+                await Task.Delay(TimeSpan.FromSeconds(30));
```

Comment:
![https://c.tenor.com/RjxBPYOLI14AAAAC/im-not-angry-im-just-disappointed.gif](https://c.tenor.com/RjxBPYOLI14AAAAC/im-not-angry-im-just-disappointed.gif)

Reply:
Unfortunately, there isn't another way to trigger the error handler without resorting to reflection hacks.

Reply:
Disposed client as you suggested.